src/components/ListItem.jsx

```diff
@@ -39,19 +41,18 @@ export function ListItem({ item, listPath }) {
 
 				await updateItem(listPath, id, { ...updatedItem });
 			} catch (error) {
-				alert(`Item was not marked as purchased`, error.message);
+				toast.error(`Item was not marked as purchased`, error.message);
 			}
 		}
 	};
 
 	const handleDeleteItem = async () => {
-		if (confirm(`Are you sure you want to delete this item?`)) {
-			try {
-				await deleteItem(listPath, id);
-			} catch (error) {
-				alert('Item was not deleted');
-			}
```

Comment:
need to add this if block back to meet previously required acceptance criteria

Reply:
done